docs/design/enums_or_ids.md

```diff
@@ -0,0 +1,129 @@
+Enums or Identifiers
+====================
+
+An *entity* is a discrete item in a set. For example, English is an entity: it is a specific instance of a language.
+
+There are two general ways to represent entities: *enumerations* and *identifiers*. An *enumeration* is a fixed, finite list of entities. An *identifier* is a universally understood space of numbers or strings, which may or may not be finite. For example, the identifier `"en"` is a language code according to a standard published by ISO.
+
+This design doc helps explain how we think about entities in ICU4X and when to use enums versus identifiers to represent them.
+
+## Decision Tree
+
+If you need to decide whether to use an enum or an identifier for a certain set of entities in ICU4X, consider the following questions:
+
+1. Is the set **fixed**?
+2. Is the set **comprehensive**? Is it inclusive of all peoples and cultures? Are there no additional entities that are not currently represented?
+3. Are there **30 or fewer items** in the set?
```

Comment:
I continue to think that the number of entries is just a proxy for the other questions, and not meaningful on its own, but this is not a hill I want to die on.